apps/services/user-profile/infra/service-portal-api.ts

```diff
@@ -23,6 +23,11 @@ export const serviceSetup = (): ServiceBuilder<'service-portal-api'> =>
         staging: 'false',
         prod: 'false',
       },
+      USER_PROFILE_WORKER_PAGE_SIZE: {
+        dev: '3000',
+        staging: '3000',
+        prod: '3000',
+      },
```

Comment:
Should I just leave this out?

Reply:
No I like the idea of processing this in pages.